pkgs/servers/tigerbeetle/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zig
+}:
+stdenv.mkDerivation rec {
+  pname = "tigerbeetle";
+  version = "unstable-2022-09-30";
+
+  src = fetchFromGitHub {
+    owner = "tigerbeetledb";
+    repo = pname;
```

Comment:
Minor nit: I would suggest using `repo - "tigerbeetle"` directly here. It reads slightly easier and will very unlikely be changed. for the package.